Makefile

```diff
@@ -20,7 +20,7 @@ test: checkstyle test-unit
 test-unit: test-bash test-python
 
 test-bash: $(BPAN)
-	prove -r $(if $v,-v )$(test)
+	prove -v -r $(if $v,-v )$(test)
```

Comment:
that doesn't make sense.
there is already a `-v` here in this line, can you see it?
but you have to enable it via a `make ... v=1`

Reply:
not showed it. and I guess you want to keep it like that. so i am closing this

Reply:
> not showed it.

?